setup.py

```diff
@@ -17,7 +17,7 @@
     include_package_data=True,
     platforms='any',
     install_requires=[
-        'elasticsearch<=1.9.0',
+        'elasticsearch>=6.0.0',
```

Comment:
Same here.  Requiring it to be >=6.0.0 is sort of limiting.  Do we really require that, or can we get away with a lower version?

Reply:
probably, but we haven't used less that v6 for FFwd, and v6 did introduce some ES API changes

Reply:
it would also mean testing a bunch of ES versions which we don't use